homeassistant/components/switcher_kis/cover.py

```diff
@@ -63,33 +74,12 @@ class SwitcherCoverEntity(SwitcherEntity, CoverEntity):
         | CoverEntityFeature.STOP
     )
 
```

Comment:
```suggestion
    _cover_id: int
```
If you add it to the base class it will resolve the warning

Reply:
It is done.